test/unit/math/prim_scalar_sig_test.cpp

```diff
@@ -670,7 +670,7 @@ TEST(PrimScalarSigTests, ldexp) {
 }
 
 TEST(PrimScalarSigTests, pow) {
-  double real_1 = 0.4;
+  using namespace std::complex_literals double real_1 = 0.4;
```

Comment:
```suggestion
  using namespace std::complex_literals;
  double real_1 = 0.4;
```

Reply:
Could we also add zeros to all numbers in complex declarations?
```c++
  std::complex<double> complex_1 = 1.0 + 2.0i;
  std::complex<double> complex_2 = 2.0 + 3.0i;
```
at
https://github.com/stan-dev/math/blob/74f2f30c9131f5bb9b112910fc09e7b258947632/test/unit/math/prim_scalar_sig_test.cpp#L677-L678

Reply:
Sure. Will do that.